addon/mixins/select.js

```diff
@@ -2,7 +2,7 @@ import Mixin from "@ember/object/mixin";
 
 export default Mixin.create({
   setLabel(value) {
-    let checkValue = value ? value : this.get('value')
+    let checkValue = this.getCheckValue(value);
```

Comment:
Could you explain what's going on around this `checkValue` and `getCheckValue` code? Just a little hard to follow on a Monday morning haha.

Reply:
I had to add this because when passing in "groupedContent" it was changing the format of what I was expecting so, I needed change whether I was using `value` or `this.get('value')`.